### PR TITLE
Feat #428 Highlight publishable parameters in parameters editor

### LINKED ISSUE #428: [Parameter Editor] Changes are not easy to keep track of
It would be nice if the user was given some sort of feedback when a value is changed. For example the value could go blue like in the IME.

### PR DESCRIPTION
**Pull request by joao4all**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
Closes #428 [Parameter Editor] Changes are not easy to keep track of

- Now publishable parameters are highlighted in the parameter table component
- Tests added

### CODE REVIEW

File: COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs [hunk 1]
```diff
@@ -89,5 +89,33 @@ protected override void OnInitialized()
...
+
+        /// <summary>
+        /// Customizes the table rows
+        /// </summary>
+        /// <param name="e">The <see cref="GridCustomizeElementEventArgs"/></param>
+        private void OnCustomizeElement(GridCustomizeElementEventArgs e)
+        {
+            if (e.ElementType == GridElementType.DataRow)
+            {
+                var isPublishable = (bool)e.Grid.GetRowValue(e.VisibleIndex, nameof(ParameterBaseRowViewModel.IsPublishable));
```
Comment (antoineatrhea) [inline COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs:101]: Is it not possible to directly get the ParameterBaseRowViewModel instance ? and then grab the IsPublishable value?

File: COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs [hunk 2]
```diff
@@ -89,5 +89,33 @@ protected override void OnInitialized()
...
+                if (isPublishable)
+                {
+                    e.CssClass = "font-weight-bold";
+                }
+            }
+
+            if (e.ElementType == GridElementType.GroupCell)
+            {
+                var elementBaseName = (string)e.Grid.GetRowValue(e.VisibleIndex, nameof(ParameterBaseRowViewModel.ElementBaseName));
+                var numberOfPublishableRowsInGroup = this.sortedCollection.Count(x => x.IsPublishable && x.ElementBaseName == elementBaseName);
```
Comment (antoineatrhea) [inline COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs:112]: for better perf, use Any() instead of Count => It will reduce computation time

**Review by antoineatrhea — CHANGES_REQUESTED** [review]
Perf' improvement

File: COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs (hunk 1 above)
Comment (joao4all) [reply COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs]: Yes, just updated the PR with this change

**Review by antoineatrhea — APPROVED** [review]

**Pull request merged**